.travis.yml

```diff
@@ -1,16 +1,29 @@
-language: python
-python:
-  - "3.7-dev"
-
 services:
-  - docker
+    - docker
 
-before_install:
-  - pip install --upgrade pip
-install:
-  - pip install codecov
+stages:
+  - test
+  - name: publish
+    if: tag IS present
 
-script:
-  - make test
-after_success:
-  - codecov
+jobs:
+  include:
+    - stage: test
+      language: python
+      python:
+        - "3.7-dev"
+      before_install:
+        - pip install --upgrade pip
+      install:
+        - pip install codecov
+      script:
+        - make test
+      after_success:
+        - codecov
+    - stage: publish
+      language: generic
+      deploy:
+          provider: script
```

Comment:
Style: looks like this block is indented by 2 extra spaces as well

Reply:
fixed